.github/workflows/dummy.yml

```diff
@@ -0,0 +1,22 @@
+name: Add a Dummy commit to trigger dev build
```

Comment:
this shows as job name under "Actions" . Can you change to "Dummy commit Scheduler" or something more meaningful please.

Reply:
Well, the action this is doing is to push a dummy commit. Scheduled execution is it's feature. So, perhaps better name could be like "Scheduled Dummy Commit to develop Branch". (which can actually be triggered manually as well, though.)